uncontrast_studies/processors/parameters_distribution_free_queries.py

```diff
@@ -92,6 +94,21 @@ def get_queryset(self):
         if len(self.suppression_methods_types):
             queryset = queryset.filter(suppression_methods__type__id__in=self.suppression_methods_types)
 
+        if len(self.outcome_types):
+            # TODO: fix this later to use the type
```

Comment:
which one of us will fix this? I don't think I know how to

Reply:
I would after you introduce a type